openshift/templates/db/db-deploy.param

```diff
@@ -7,11 +7,14 @@ NAME=identity-kit-db
 APP_NAME=identity-kit
 APP_GROUP=identity-kit
 SUFFIX=-bc
+BACKUP=true
 ROLE=db
+API_ROLE=controller
+NAMESPACE_NAME=a99fd4
 IMAGE_NAMESPACE=a99fd4-tools
 SOURCE_IMAGE_NAME=issuer-kit-db
 TAG_NAME=test
-MONGODB_DATABASE=identity_kit_bc_db
+MONGODB_DATABASE=identity_kit_db
```

Comment:
Did this change with the new deployments? I see the data from the backup is there, just confirming what's up

Reply:
identity_kit_db was the name of the database in OCP3 so I decided to keep it consistent

Reply:
Ok, that seems like a reasonable thing to do. Thanks for clarifying!